k8s/upgrades/0.8.0-0.8.1/jiva_volume_upgrade.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+set -e
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+function usage() {
+    echo 
+    echo "Usage:"
+    echo 
+    echo "$0 <pv-name>"
+    echo 
+    echo "  <pv-name> Get the PV name using: kubectl get pv"
+    exit 1
+}
+
+function setDeploymentRecreateStrategy() {
+    dns=$1 # deployment namespace
+    dn=$2  # deployment name
+    currStrategy=`kubectl get deploy -n $dns $dn -o jsonpath="{.spec.strategy.type}"`
+
+    if [ $currStrategy = "RollingUpdate" ]; then
+       kubectl patch deployment --namespace $dns --type json $dn -p "$(cat patch-strategy-recreate.json)"
+       rc=$?; if [ $rc -ne 0 ]; then echo " Upgrade failed | ERROR: $rc"; exit; fi
+       echo "Deployment upgrade strategy set as recreate"
+    else
+       echo "Deployment upgrade strategy was already set as recreate"
+    fi
+}
+
+if [ "$#" -ne 1 ]; then
+    usage
+fi
+
+pv=$1
+replica_node_label="openebs-jiva"
+
+# Check if pv exists
+kubectl get pv $pv &>/dev/null;check_pv=$?
+if [ $check_pv -ne 0 ]; then
+    echo "$pv not found";exit 1;    
+fi
+
+# Check if CASType is jiva
+cas_type=`kubectl get pv $pv -o jsonpath="{.metadata.annotations.openebs\.io/cas-type}"`
+if [ $cas_type != "jiva" ]; then
+    echo "Jiva volume not found";exit 1;
+fi
+
+pvc=`kubectl get pv $pv -o jsonpath="{.spec.claimRef.name}"`
+ns=`kubectl get pv $pv -o jsonpath="{.spec.claimRef.namespace}"`
+sc_name=`kubectl get pv $pv -o jsonpath="{.spec.storageClassName}"`
+sc_res_ver=`kubectl get sc $sc_name -n $ns -o jsonpath="{.metadata.resourceVersion}"`
+
+################################################################# 
+# STEP: Generate deploy, replicaset and container names from PV #
+#                                                               #
+# NOTES: Ex: If PV="pvc-cec8e86d-0bcc-11e8-be1c-000c298ff5fc"   #
+#                                                               #
+# ctrl-dep: pvc-cec8e86d-0bcc-11e8-be1c-000c298ff5fc-ctrl       # 
+#################################################################
+
+c_dep=$(echo $pv-ctrl)
```

Comment:
most of the review comments of cstor volume upgrade script will apply here

Reply:
will make the changes ASAP